shap/explainers/_tree.py

```diff
@@ -155,7 +158,17 @@ def __init__(
             Currently the "probability" and "log_loss" options are only
             supported when ``feature_perturbation="interventional"``.
 
+        approximate : bool
+            Deprecated, will be removed in a version v.0.48.0. Please use the approximate argument in the
+            ``shap_values`` or ```__call__`` methods instead.
+
         """
+        if approximate is not DEPRECATED_APPROX:
+            warnings.warn(
+                "The approximate argument has been deprecated and will be removed in a future version. "
+                "Please use the approximate argument in the TreeExplainer constructor instead.",
```

Comment:
They should be using the approximate argument in the shap_values or __call__ methods, not the constructor.

Reply:
done